tests/test_devices/test_update_device.py

```diff
@@ -19,3 +20,28 @@ def test_update_device_with_non_existant_id(self):
             query_with_non_existant_id,
             "DeviceId not found"
         )
+
+    def test_database_connection_error(self):
+        """
+        test a user friendly message is returned to a user when database
+        cannot be reached
+        """
+        BaseTestCase().tearDown()
+        CommonTestCases.admin_token_assert_in(
+            self,
+            update_device_query,
+            "The database cannot be reached"
+        )
+
+    def test_update_device_without_devices_model(self):
+        """
+        test if a user can update a device when the device model is missing
```

Comment:
Can a user update a device when the device model is missing?

Reply:
Done @Bonifase